src/api/api_routes.js

```diff
@@ -15,6 +15,7 @@ module.exports = [
     ['POST', '/v1/api/identity/blockTimestamp', 'IdentityController@blockTimestamp'],
     ['POST', '/v1/api/identity/open', 'IdentityController@openLink'],
     ['POST', '/v1/api/identity/register', 'IdentityController@registerIdentity'],
+    ['GET', '/v1/api/identity/:service/:domain', 'IdentityController@resolveDomain'],
```

Comment:
I think it makes sense to add this endpoint to the `identity` controller. But if you think there's a better place for it, please let me know.